contrib/charts/dragonfly/values.yaml

```diff
@@ -133,7 +133,7 @@ tls:
   #   HLunol88AeTOcKfD6hBYGvcRfu5NV29jJxZCOBfbFQXjnNlnrhRCag==
   #   -----END RSA PRIVATE KEY-----
 
-# If enabled will set DFLY_PASSOWRD environment variable with the specified existing secret value
+# If enabled will set DFLY_PASSWORD environment variable with the specified existing secret value
```

Comment:
We have deprecated DFLY_PASSWORD, isn't that right ? @romange

Reply:
Yes, a year ago, its now `DFLY_requirepass`. We can update it later separately